src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -69,27 +84,43 @@
     }
 
     public static @Nonnull GitHub connect(@CheckForNull String apiUri, @CheckForNull StandardCredentials credentials) throws IOException {
```

Comment:
The method creates a `GitHub` object from the connection parameters, typically a user name and a personal access token.